lib/static/new-ui/store/selectors.ts

```diff
@@ -18,3 +18,5 @@ export const getResults = (state: State): Record<string, ResultEntity> => state.
 export const getImages = (state: State): Record<string, ImageEntity> => state.tree.images.byId;
 
 export const getIsInitialized = (state: State): boolean => state.app.isInitialized;
+export const getIsStaticImageAccepterEnabled = (state: State): boolean => state.config.staticImageAccepter.enabled;
```

Comment:
Dont check `is static image accepter enabled` from config.
Check it from `state.staticImageAccepter.enabled` instead.

staticImageAccepter is being initialized here: https://github.com/gemini-testing/html-reporter/blob/master/lib/static/modules/reducers/static-image-accepter.js#L10-L14

And it does only when [staticImageAccepter.checkIsEnabled](https://github.com/gemini-testing/html-reporter/blob/f70e78b569773ac61cc814d342fcb25b6c6afa54/lib/static/modules/static-image-accepter.ts#L36) returns true.

The difference between `state.staticImageAccepter.enabled` and `state.config.staticImageAccepter.enabled` occurs in these situations, where `state.staticImageAccepter.enabled` would be `false`:
- gui mode. `state.staticImageAccepter.enabled` is false in GUI;
- invalid config. If config is invalid, static image accepter is turned off.

At the same time, `state.config.staticImageAccepter.enabled` has the value which user set in html-reporter config to be

Reply:
In fact, that was the reason i created `state.staticImageAccepter.enabled` in the first place, so there would be one single point where we could check "is there working static image accepter" and don't worry about cases, where "yes, it is enabled in config, but currently is disabled because of gui"

Reply:
IMO having two `enabled` fields in state is pretty misleading, I'd never figured the difference if you didn't tell. The `staticImageAccepter.enabled` could be implemented as a selector named something like "shouldGuiniBeRendered" (because it can be derived from the rest of the state).

For now, changed to the field you recommended to use

Reply:
> misleading

Yes, but "config" is just plugin config and I think it should not be modified. It's just what user set it to be.

I don't think there is anything we can do about it